player.py

```diff
@@ -10,16 +13,17 @@ def __init__(self, num, color=""):
         self.figures = [Figure]
         self.playing = False
         self.undeployed = True
-        self.result = ""
+        self.result = 0
         self.rolls = []
         self.ai = False
         self.tactic = Tactic()
 
 
 # definování hráčů
-player1 = Player(1)
-player2 = Player(2)
-player3 = Player(3)
-player4 = Player(4)
-player5 = Player(5)
-player6 = Player(6)
+players = []
+player_traits = []
+for i in range(0, len(figures)):
+    player_traits.append([figures[i], figures[i][0].color, "[" + figures[i][0].language_color[0] + "]" +
+                         figures[i][0].language_color[1:], figures[i][0].language_color[0]])
+for i in range(1, len(player_traits) + 1):
```

Comment:
I think it is much more idiomatic to add one inside the loop, like this:
```suggestion
for i in range(len(player_traits)):
```